.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
you're not using matrix, remove the variable referencing it. note that you can omit the name field altogether, I would, it doesn't provide any additional useful information (here and in the other actions too)

Reply:
OK